chapters/03_queueing_model/01_discrete_event_simulation/main.tex

```diff
@@ -0,0 +1,281 @@
+\subsection{Discrete Event Simulation}
+
+Discrete event simulation (DES) is currently one of the most popular modelling
+techniques.
```

Comment:
Try and avoid sentences like this that are a bit vague. Depending on what you mean by "modelling" this could be incorrect at best.

```suggestion

```